test/matchers/equals.spec.js

```diff
@@ -0,0 +1,38 @@
+var EqualMatcher = require('../../lib/matchers/equals');
+
+describe('equals matcher', function() {
+
+  it('fails when expectedValue is undefined', function() {
+    new EqualMatcher().match('', undefined).should.have.error(/should be defined/);
+  });
+
+  it('fails when expectedValue is an object', function() {
+    new EqualMatcher({}).match('', {}).should.have.error(/should be a primitive/);
```

Comment:
Could we simply say `should equal {expectedValue}`? Otherwise people might think any primitive is OK.